2013/P_U1/main.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+using namespace std;
+
+struct Uzsakovas{
+    string pavadinimas;
+    int x;
+    int y;
+    int atstumas(){
+        return abs(x)+abs(y)+abs(x)+abs(y);
+    }
+};
+
+void skaitymas(int& kilometrazas, vector<Uzsakovas>& uzsakovai){
+    ifstream data("U1.txt");
+    int n;
+    data >> n;
+    data >> kilometrazas;
+    for(int i = 0; i<n; i++){
+        data.ignore();
+        char ch[11];
+        data.read(ch, 10);
+        ch[10] = '\0';
+        Uzsakovas tempUzsakovas;
+        tempUzsakovas.pavadinimas = ch;
+        data >> tempUzsakovas.x;
+        data >> tempUzsakovas.y;
+        uzsakovai.push_back(tempUzsakovas);
+    }
+    data.close();
+}
+
+void skaiciavimai(int& likoUzsakovu, int& kiekNuvaziavo, string& paskutinis, int kilometrazas, vector<Uzsakovas> uzsakovai){
```

Comment:
Comment can be added to explain what function calculates from a task perspective.